src/main/java/com/kodilla/ecommercee/UserController.java

```diff
@@ -1,4 +1,28 @@
 package com.kodilla.ecommercee;
 
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/users")
 public class UserController {
+
+    @PostMapping("/createUser")
+    public GenericEntity createUser(){
+        return new GenericEntity();
+    }
+
+    @GetMapping("/accessDenied")
+    public String accessDenied(Long userId){
```

Comment:
Jakie jest założenie metody accessDenied?

Reply:
Odmówienie dostępu dla konkretnego użytkownika (zablokowanie użytkownika, nie wiedziałam jak to inaczej zrobić).

Reply:
Ok, zamotałam się. Chodziło mi o odmowę dostępu dla konkretnego użytkownika. Ale teraz widzę, że to raczej nie powinno być na poziomie kontrolera użytkownika.